tools/misc/browse_dataset.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import collections
```

Comment:
from collections import Sequence

Reply:
ok i usually import modules instead of classes/objects. but.... sure! changing...

Reply:
changed